docs/elements.md

```diff
@@ -190,6 +190,47 @@ Two grading methods are available when using `partial-credit="true"`:
 
 -----
 
+## `pl-dropdown` element
+
+Select the correct answer from a drop-down **select** menu list of potential answers. The potential options are listed in the inner HTML of a <pl-answer></pl-answer> element (ie. <pl-answer>Possible Answer 1</pl-answer>).
+
+#### Sample Element
+
+![](elements/pl-dropdown.png)
+
+**question.html**
+
+```html
+An <pl-dropdown sort="random" correct-answer="socrates">
```

Comment:
Is this `random` attribute implemented? It isn't documented below.

Reply:
Ah! I see what you mean now. It is implemented, so I will document it. Thanks!